src/pivot-table/components/icons/styles.ts

```diff
@@ -0,0 +1,7 @@
+import { HEADER_ICON_SIZE } from "../../constants";
+import { baseFlex } from "../shared-styles";
+
+export const headerPlusMinusIconWrapperStyle = {
+  width: HEADER_ICON_SIZE,
```

Comment:
`HEADER_ICON_SIZE` is 12px but the icons are 13px. That is likely why the rendering tests are failing with "fuzzy" icons

Reply:
will update it soon